### PR TITLE
Repo Cleanup

### PR DESCRIPTION
**Pull request by rayzru**
It's not necessary to keep compiled files.
Added .gitignore options for map and compiled files
Added postinstall action in package.json

### CODE REVIEW

**Comment by rayzru** [comment]
Sorry, but i think i have added my previous PR changes into this one.

File: src/exif.ts [hunk 1]
```diff
@@ -1,21 +1,32 @@
+export interface Fraction {
+    numerator: number;
+    denominator: number;
+}
+
+declare Fraction: {
```
Comment (cstefanache) [inline src/exif.ts]: Getting error: modifiers are not allowed

File: src/exif.ts [hunk 2]
```diff
@@ -588,10 +564,10 @@ export class Exif {
                 }
 
             case 5:    // rational = two long values, first is numerator, second is denominator
-                if (numValues == 1) {
+                if (numValues === 1) {
                     numerator = file.getUint32(valueOffset, !bigEnd);
                     denominator = file.getUint32(valueOffset + 4, !bigEnd);
-                    val = Number(numerator / denominator);
+                    val = new FractionNumber(numerator / denominator);
```
Comment (cstefanache) [inline src/exif.ts]: should this be Fraction(numerator/denominator) ?

File: src/exif.ts (hunk 1 above)
Comment (cstefanache) [inline src/exif.ts]: Error at tsc:
 src/exif.ts(6,9): error TS1005: ';' expected.
src/exif.ts(7,16): error TS1109: Expression expected.
src/exif.ts(7,22): error TS1005: ';' expected.
src/exif.ts(8,12): error TS1109: Expression expected.
src/exif.ts(8,18): error TS1005: ';' expected.

**Comment by cstefanache** [comment]
Conflict should be ignore - accept your version with Fraction object (I just released a new Number) version as a hotfix

**Comment by rayzru** [comment]
Phew! So much commits. 
Finally, I got a stable state of your cool component converted to pure TS with types interfaces and stuff.

My current project is restricted with TSLint rules, so I have to do this.
Sorry. ^_^

**Comment by cstefanache** [comment]
No problem ... getting into code review ... adapting everything and releasing a 6.2 with your changes. Special thanks

**Comment by cstefanache** [comment]
Planning to merge it today.

**Comment by rayzru** [comment]
Sorry, I have to ask you. I didn't realise yet, how to upload cropped image with angular2? I saw  suggestions related to `ng2-file-upload` by @valor-software but... u know, that too complicated for me to merge two components into single one.

Also, I understand that we got encoded `base64` data as a result, So I guess, we can just pass that data with multipart headers as regular POST data?

Thank you in advance.

**Comment by cstefanache** [comment]
Usually this is intended to work with REST APIs with Methods that support BODY data such as POST,PUT etc. Since you are not submitting a form as multipart it can be used to send an entire JSON with information to the server - one of the attr can be base64 image data

**Comment by rayzru** [comment]
Sure, i have REST API accepting multipart-form-data. So is it possible to send cropped `data` as regular multipart POST?

**Comment by cstefanache** [comment]
Getting: (need to investigate why)
image

**Comment by rayzru** [comment]
Try add `export` here:
`declare var Fraction: {` => `export declare var Fraction: {`

**Comment by cstefanache** [comment]
Was there a good reason to change the runtime test app? The examples there show how to use both squared and rounded cropper

**Comment by cstefanache** [comment]
Merged and Released 0.6.4

**Comment by cstefanache** [comment]
Thanks!

**Comment by cstefanache** [comment]
Can you please give a test-run

**Comment by rayzru** [comment]
> Was there a good reason to change the runtime test app? The examples there show how to use both squared and rounded cropper

No, just for a huge amount of tests and debug code i have to shrink it this way.

**Comment by rayzru** [comment]
Ok, I try to do test-run this evening.

**Pull request merged**